spec/rubocop/cop/rspec/file_path_spec.rb

```diff
@@ -172,6 +172,15 @@
     RUBY
   end
 
+  it 'uses relative path' do
+    allow(Dir).to receive(:pwd).and_return('/home/foo')
+    allow(File).to receive(:absolute_path).and_return('/')
```

Comment:
It's quite ok, but what do you think of stubbing `RuboCop::PathUtil.relative_path` directly? It's implementation-specific, but why not?

Reply:
True. I changed to stub `RuboCop::PathUtil.relative_path` instead.